networkx/generators/joint_degree_seq.py

```diff
@@ -0,0 +1,283 @@
+"""Generate graphs with a given joint degree 
+"""
+#    BSD license.
+import random
+
+import networkx as nx
+
+__author__ = "\n".join(['Minas Gjoka <minas.gjoka@gmail.com>'])
```

Comment:
This is not required, author information is preserved in the git commit and the file header above.